packages/node/test/integration/unidirectional-transfer.ts

```diff
@@ -0,0 +1,51 @@
+import { AppABIEncodings } from "@counterfactual/types";
```

Comment:
This and some other files are helpers / utils for a specific set of contracts and should not be in the `test/integration/*` path, maybe put in a subfolder